src/node/in_memory.rs

```diff
@@ -74,7 +75,7 @@ pub const TEST_NODE_NETWORK_ID: u32 = 260;
 /// L1 Gas Price.
 pub const L1_GAS_PRICE: u64 = 50_000_000_000;
 /// L2 Gas Price (0.25 gwei).
```

Comment:
Let's update this comment to match

Reply:
my bad - this change to l2 gas price should go in separate PR